pkg/api/dataproxy.go

```diff
@@ -33,8 +34,10 @@ func (hs *HTTPServer) ProxyDataSourceRequest(c *models.ReqContext) {
 		return
 	}
 
+	proxyPath := getProxyPath(c.Req.URL.EscapedPath())
+
 	// macaron does not include trailing slashes when resolving a wildcard path
-	proxyPath := ensureProxyPathTrailingSlash(c.Req.URL.Path, c.Params("*"))
+	proxyPath = ensureProxyPathTrailingSlash(c.Req.URL.Path, proxyPath)
```

Comment:
I think `ensureProxyPathTrailingSlash` is no longer needed? Previously `c.Params()` would remove the trailing slash, but afaik `c.Req.URL.EscapedPath()` does not

Reply:
Good catch :+1: